benches/bench.rs

```diff
@@ -834,10 +832,11 @@ fn benchmark_remap(c: &mut Criterion) {
 }
 
 fn random_lines(size: usize) -> impl Iterator<Item = String> {
-    let mut rng = SmallRng::from_rng(thread_rng()).unwrap();
+    let rng = SmallRng::from_rng(thread_rng()).unwrap();
 
     std::iter::repeat(()).map(move |_| {
-        rng.sample_iter(&Alphanumeric)
+        rng.clone()
```

Comment:
@fanatid Just noting that cloning the rng here will result in creating identical lines as it uses the same seed. I'm fixing this by removing this function in-lieu of the `random_lines` in `test_utils` but just calling to your attention that I think we typically don't want to clone rngs.